.travis.yml

```diff
@@ -15,7 +15,7 @@ before_script:
 script:
   - make travis-test-with-coverage
 after_success:
-  - if [ "$COVERALLS_TOKEN" ]; then goveralls -coverprofile=profile.cov -service=travis-ci -repotoken $COVERALLS_TOKEN; fi
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
This is in the Travis script only, so the target vector is quite small. Committing the specific script version may cause random breaks if their backend becomes de-sync'd.